.github/workflows/wiki_label.yml

```diff
@@ -0,0 +1,47 @@
+name: 'PR Wiki Label'
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  wiki-label:
+    runs-on: ubuntu-latest
+    if: github.event.issue.pull_request && github.event.comment.body == '!wiki_label'
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Check commenter authorization
+        id: check-auth
+        run: |
+          IFS=',' read -ra WIKI_MANAGERS <<< "${{ vars.WIKI_MANAGERS }}"
+          COMMENTER="${{ github.event.comment.user.login }}"
+          if [[ " ${WIKI_MANAGERS[@]} " =~ " ${COMMENTER} " ]]; then
+            echo "authorized=true" >> $GITHUB_OUTPUT
+          else
+            echo "authorized=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Add Wiki label and thumbs up reaction
+        if: steps.check-auth.outputs.authorized == 'true'
+        run: |
+          gh pr edit ${{ github.event.issue.number }} --add-label "Requires Wiki Update"
+          echo "Label 'Requires Wiki update' added to PR #${{ github.event.issue.number }}"
```

Comment:
Do we need this line for when it is added, i think it might be clear enough when the label is just added?

Reply:
This is for the workflow itself, I removed the comments. `echo` here  is what will show up when you looking at the workflow actions themselves. But yeah, the consensus was that the thumbs up and down and the label being added would just be enough.